server/src/weaverbird/pipeline/steps/filter.py

```diff
@@ -13,3 +13,8 @@ class FilterStep(BaseStep):
 
 class FilterStepWithVariables(FilterStep, StepWithVariablesMixin):
     condition: ConditionWithVariables  # type:ignore[assignment]
+
+    def model_dump(self, *, exclude_none: bool = False, **kwargs) -> dict[str, Any]:
+        # we don't want to drop fields with value=None like:
+        #     {'column': 'nullable_name', 'operator': 'eq', 'value': None}
+        return super().model_dump(exclude_none=exclude_none, **kwargs)
```

Comment:
isn't `exclude_none=False` already the default behavior ? https://docs.pydantic.dev/latest/api/base_model/#pydantic.BaseModel.model_dump

Reply:
we have it overriden with exclude_none=True in BaseStep

Reply:
:bulb: